nntrainer/src/util_func.cpp

```diff
@@ -147,4 +147,49 @@ float reluPrime(float x) {
     return 1.0;
   }
 }
+
+// This is 2D zero pad
+// TODO : Optimize for multi dimention padding
+Tensor zero_pad(int batch, Tensor in, unsigned int const *padding) {
+  unsigned int c = in.getDim().channel();
+  unsigned int h = in.getDim().height();
+  unsigned int w = in.getDim().width();
+
+  unsigned int height_p = h + padding[0] * 2;
+  unsigned int width_p = w + padding[1] * 2;
+
+  unsigned int height_p_h = h + padding[0];
+  unsigned int width_p_h = w + padding[1];
+
+  Tensor output(1, c, height_p, width_p);
+
+  if (padding[0] != 0 || padding[1] != 0) {
```

Comment:
I think this condition does nothing. The below for-loops are already checking `padding[0]` and `padding[1]`.

Reply:
done. Thanks